runtime/src/common/bytes.rs

```diff
@@ -11,7 +11,7 @@
 macro_rules! impl_bytes {
     ($name:ident, $size:expr, $doc:expr) => {
         #[doc=$doc]
-        #[derive(Clone, Copy)]
+        #[derive(Clone, Copy, zeroize::Zeroize)]
```

Comment:
Do we want to zeroize all, even public keys and hashes?

Reply:
No, this only implements `Zeroize` which provides the `zeroize` method. It does not automatically zeroize (e.g. on drop).